apps/src/templates/sectionProgress/StudentProgressDetailCell.jsx

```diff
@@ -32,6 +32,7 @@ export default class StudentProgressDetailCell extends Component {
             selectedStudentId={this.props.studentId}
             pairingIconEnabled={true}
             stageExtrasEnabled={this.props.stageExtrasEnabled}
+            progressView={true}
```

Comment:
a better name for this would be something which tells the component what to do, such as  `showAssessments`, `showAssessmentCheck`, `highlightAssessments`, etc. here and everywhere `progressView` is used. I know this seems minor, but this kind of "declarative" naming can really improve readability.